docs/integrations/sources/slack.md

```diff
@@ -137,6 +137,7 @@ It is recommended to sync required channels only, this can be done by specifying
 
 | Version | Date       | Pull Request                                             | Subject                                                                             |
 |:--------|:-----------|:---------------------------------------------------------|:------------------------------------------------------------------------------------|
+| 0.3.4   | 2023-10-06 | [31134](https://github.com/airbytehq/airbyte/pull/31134) | Update CDK and remove non iterable return from records                              |
```

Comment:
```suggestion
| 0.3.4   | 2023-10-06 | [31134](https://github.com/airbytehq/airbyte/pull/31134) | Update CDK and remove no-interator return from records                              |
```
.. I may have messed up the table with this suggestion so take it with a grain of salt lol

Reply:
I.... do not like this change.

But! I loved all the others 😛